### DISCUSSION TITLE
Can I use vercel/pkg to package the whole project?

### DISCUSSION THREAD

**Discussion by sohaha** [nuxt/nuxt#22294] [Ideas]
Or is there another way to use server-side rendering mode without nodejs installed?

**Comment by MertcanYuzer** [comment]
Hello, I want to do the same thing, did you manage to do it?

**Reply by sohaha** [reply]
Sorry, we don't have any at the moment, we can only give up.

**Reply by MertcanYuzer** [reply]
It seems like we have no choice but to wait for help from someone in the Core team.
